docs/sources/write/shortcodes/index.md

```diff
@@ -412,6 +412,71 @@ This isn't rendered.
 
 This is rendered after the ignore.
 
+## Docs/openapi/info
+
+Display information about an OpenAPI 3.0+ specification, use either the `url` or `data` parameter to specify an OpenAPI specification.
+
+| Parameter | Description                                                                                       | Required |
+| --------- | ------------------------------------------------------------------------------------------------- | -------- |
+| `url`     | The URL of the OpenAPI specification to fetch.                                                    | no       |
+| `data`    | The filename of the OpenAPI specification to use from the website `data/docs/openapi/` directory. | no       |
+
+To fetch a remote specification from a URL:
+
+```markdown
+{{/*% docs/openapi/info url="<SPECIFICATION_URL>" %*/}}
```

Comment:
https://github.com/grafana/writers-toolkit/pull/833#pullrequestreview-2279793232

For example,
```suggestion
{{</* docs/openapi/info url="<SPECIFICATION_URL>" */>}}
```

The same applies for all the snippets